src/pivotal-ui/components/header/package.json

```diff
@@ -4,5 +4,5 @@
     "@npmcorp/pui-css-drop-down-menu": "^1.3.0",
     "@npmcorp/pui-css-typography": "^4.0.0"
   },
-  "version": "2.2.2"
+  "version": "3.0.0-0"
```

Comment:
Yep. You can have one set of digits in there, pretty much any way. Just not two dashes.